src/test/objectstore/Allocator_test.cc

```diff
@@ -187,6 +187,49 @@ TEST_P(AllocTest, test_alloc_failure)
   }
 }
 
+TEST_P(AllocTest, test_alloc_hint_bmap)
+{
+  if (GetParam() == std::string("stupid")) {
```

Comment:
The stupid allocator was not passing all new hint test cases. So added this to bypass it for stupid allocator.